packages/@react-spectrum/dnd/docs/dragAndDrop.mdx

```diff
@@ -0,0 +1,350 @@
+{/* Copyright 2022 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License. */}
+
+import {Layout} from '@react-spectrum/docs';
+export default Layout;
+
+import ChevronRight from '@spectrum-icons/workflow/ChevronRight';
+import docs from 'docs:@react-spectrum/dnd';
+import dndDocs from 'docs:@react-types/shared/src/dnd.d.ts';
+import {FunctionAPI, PageDescription, TypeContext, InterfaceType} from '@react-spectrum/docs';
+import {Keyboard} from '@react-spectrum/text';
+
+---
+category: Concepts
+keywords: [drag and drop, dnd]
+after_version: 3.0.0
+order: 6
+---
+
+# Drag and Drop
```

Comment:
Open to opinions on whether or not enough information is covered. My goal was to make it introductory level and not to get too deep into the details since that might put readers off